examples/game_tech/agones-game-controller/README.md

```diff
@@ -0,0 +1,216 @@
+ - Deploy EKS Cluster with one managed node group in an VPC
+ - Deploy Agones Kubernetes Controller using Helm Providers
+ - Deploy a simple gaming server and test the application
+
+# What is Agones
+
+Agones is an Open source Kubernetes Controller with custom resource definitions and is used to create, run, manage and scale dedicated game server processes within Kubernetes clusters using standard Kubernetes tooling and APIs.
+This model also allows any matchmaker to interact directly with Agones via the Kubernetes API to provision a dedicated game server
+
+# What is GameLift
+
+Amazon GameLift enables developers to deploy, operate, and scale dedicated, low-cost servers in the cloud for session-based, multiplayer games.
+Built on AWS global computing infrastructure, GameLift helps deliver high-performance, high-reliability, low-cost game servers while dynamically scaling your resource usage to meet worldwide player demand.
+
+# Deployment Steps
+
+Execute **Step1** only when you are running this workshop using AWS EventEngine. Workshop conducted by the AWS teams at Customer sites
+
+## Step1: Login to Event Engine as advised by AWS team. You may need a temporary AWS Account token for this step
+
+    https://www.eksworkshop.com/020_prerequisites/aws_event/portal/
+
+## Step2: Create [Cloud9 IDE in AWS Environment](https://www.eksworkshop.com/020_prerequisites/workspace/#region-1) and install Pre-requisites
+
+Use the following link to install Cloud9 IDE and all the pre-requisites Kubernetes tools
+
+    https://www.eksworkshop.com/020_prerequisites/workspace/#region-1
+
+### Step2.1: Install [Kubernetes Tools](https://www.eksworkshop.com/020_prerequisites/k8stools/)
+    https://www.eksworkshop.com/020_prerequisites/k8stools/
+
+### Step2.2: Create [IAM Role for Workspaces](https://www.eksworkshop.com/020_prerequisites/iamrole/)
+    https://www.eksworkshop.com/020_prerequisites/iamrole/
+
+### Step2.3: Attach IAM role to Cloud9 IDE and update IAM settings
+    https://www.eksworkshop.com/020_prerequisites/ec2instance/
+    https://www.eksworkshop.com/020_prerequisites/workspaceiam/
+
+## Step3: Install `aws-iam-authenticator` in Cloud9 IDE
+     curl -o aws-iam-authenticator https://amazon-eks.s3.us-west-2.amazonaws.com/1.19.6/2021-01-05/bin/linux/amd64/aws-iam-authenticator
+
+     openssl sha1 -sha256 aws-iam-authenticator
+
+     chmod +x ./aws-iam-authenticator
+
+     mkdir -p $HOME/bin && cp ./aws-iam-authenticator $HOME/bin/aws-iam-authenticator && export PATH=$PATH:$HOME/bin
+
+     echo 'export PATH=$PATH:$HOME/bin' >> ~/.bashrc
+
+     Verify the installation
+
+     aws-iam-authenticator help
+
+## Step4: Install Terraform in Cloud9 IDE
+
+    sudo yum remove -y terraform
+
+    sudo yum install -y yum-utils
+
+    sudo yum-config-manager --add-repo https://rpm.releases.hashicorp.com/AmazonLinux/hashicorp.repo
+
+    sudo yum -y install terraform-1.0.1
+
+    sudo terraform version
+
```

Comment:
These steps seem like they came from a workshop? We are not describing these in other examples, so perhaps remove them from here as well.

Reply:
It was part of workshop delivered to customer. I will remove this now